terraform/keys.tfvars

```diff
@@ -0,0 +1,2 @@
+access_key = "XXXXXXXXXXXXXXXXXXX"

```

Comment:
What would you suggest instead of having keys in this file? Is there a better approach?

Reply:
environmental variables like TF_VAR_access_key can be used, but I added the .tfvars file to show which variables should be created